api_calls/reference/participants/inbox_write-scope-response.json

```diff
@@ -0,0 +1,32 @@
+{
+  "links": {
+    "participants.conversation": "http://test.host/api/v3/inbox/conversations/{participants.conversation}",
+    "participants.client": "http://test.host/api/v3/clients/{participants.member.id}",
+    "participants.host": "http://test.host/api/v3/hosts/{participants.member.id}"
+  },
+  "participants": [
+    {
+      "links": {
+        "conversation": 3,
+        "member": {
+          "id": 5,
+          "type": "Host"
```

Comment:
hmm, in that case we need docs for /hosts endpoint but that could be a separate PR